scripts/control_device.py

```diff
@@ -231,7 +231,7 @@
     choices=possible_fan_level.keys(),
     required=False,
     help=f"thermostat fan level, possible values: {possible_fan_level}",
-    default=None,
+    default="",
```

Comment:
based on the enum this possible values set is constructed from,
an empty string is not a valid option, why do we have it as a default value?

Reply:
because it is required for the control_thermostat function.

The argparse module won't assign value for this argument if its not required and no default value specified

Reply:
see above https://github.com/TomerFi/aioswitcher/pull/492#discussion_r912333604

Reply:
see above https://github.com/TomerFi/aioswitcher/pull/492#discussion_r912333644